spec/services/idv/session_spec.rb

```diff
@@ -193,41 +193,64 @@
           subject.applicant = Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE.with_indifferent_access
         end
 
-        it 'sets profile to pending in person verification' do
-          subject.create_profile_from_applicant_with_password(user.password, is_enhanced_ipp)
-          profile = subject.profile
-
-          expect(profile.activated_at).to eq nil
-          expect(profile.active).to eq false
-          expect(profile.in_person_verification_pending?).to eq(true)
-          expect(profile.fraud_review_pending?).to eq(false)
-          expect(profile.gpo_verification_pending_at.present?).to eq false
-          expect(profile.initiating_service_provider).to eq nil
-          expect(profile.verified_at).to eq nil
-
-          pii_from_session = Pii::Cacher.new(user, user_session).fetch(profile.id)
-          expect(pii_from_session).to_not be_nil
-          expect(pii_from_session.ssn).to eq(Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE[:ssn])
+        context 'when the USPS enrollment is successful' do
+          let(:profile) { subject.profile }
+          let(:pii_from_session) { Pii::Cacher.new(user, user_session).fetch(profile.id) }
+
+          before do
+            allow(UspsInPersonProofing::EnrollmentHelper).to receive(:schedule_in_person_enrollment)
+            subject.create_profile_from_applicant_with_password(user.password, is_enhanced_ipp)
```

Comment:
So I moved this to before block because I felt like it would dry up the it blocks a bit. My understanding is that before blocks are for both setup and execution of code under test. That being said I don't mind putting subject call in the it block it, it just feels like a lot of unnecessary duplication.